feature/registration-signin.md

```diff
@@ -0,0 +1,34 @@
+# Registration and Sign In
+
+Although Cambiatus uses EOS, we have a Sign Up/Sign In process that is a little different from other blockchain apps. First of all we try to abstract public and private key pairs from the user.
+
+## Basic registration steps
+
+If the user isn't trying to register from an Invitation, we use the basic re/gistration process. It all begins by the user clicking on `Don't have an account yet? Create now` on the Login page. From there the user will be taken to the registration page, where he/she will have to fill in 3 infos: Full name, account name (which is an [EOS account](integration/accounts.html)) and an email address. 
+
+The frontend will generate this kind of request:
+
+```graphql
+mutation {
+  signUp3423593742: signUp(input: 
+    {
+      account: "minminminbb4", 
+      email: "a@a",
+      name: "Test Min",
+      publicKey: "EOS6ha8YSp5BHwHxmXW76M8zC7mCCRYNvbVw7Ja3grxDhYV6UKfjw"
+    }) {
+    status955176064: status
+    reason3832528868: reason
+  }
+}
+```
+
+Now the backend will take over. Roughly the backend will do the following steps:
+
+- Try to find if the `account` is already registred;
```

Comment:
>  already registred;

already registered

Reply:
Thanks for noticing, I'll fix. You can do so too! Try clicking on that button and add a suggestion, its easier to apply them and easier for you to point it out!

<img width="642" alt="Captura de Tela 2020-11-02 às 13 08 13" src="https://user-images.githubusercontent.com/1082127/97890604-7da96a80-1d0c-11eb-9a95-57819ccd0128.png">

Reply:
```suggestion
- Try to find if the `account` is already registered;
```